api/justfile

```diff
@@ -83,7 +83,7 @@ collectstatic:
 #########
 
 # Run API tests inside the Docker container
-test *args: _all-up
+test *args:
```

Comment:
I think `test` needs to depend on `wait-up`, hence the merge conflict. I think using out this file as it exists on the `main` branch should be fine.

Reply:
Thanks Dhruv!